lib/testsuite/index.js

```diff
@@ -1,5 +1,5 @@
 const AssertionError = require('assertion-error');
-const {By, Key, locateWith, withTagName} = require('selenium-webdriver');
+const {By, Key, locateWith, withTagName, Browser} = require('selenium-webdriver');
```

Comment:
This import is not required

Reply:
fixed in [081717f](https://github.com/nightwatchjs/nightwatch/pull/3609/commits/081717f83b6e3fc89977e7206f4cebc9870a1077)